### PR TITLE
How to migrate from Py2 to Py3

### PR DESCRIPTION
**Pull request by jvanderaa**
The commands done to migrate Ubuntu from Py2 to Py3.

### Fixes:

Provides documentation of migrating from Py2 to Py3 on Ubuntu 16.04.

### CODE REVIEW

File: docs/installation/Migrating-to-Python3 [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Migration
+Remove Python 2 packages
+```no-highlight
+# apt-get remove --purge -y python-dev python-pip libxml2-dev libxslt1-dev libffi-dev graphviz libpq-dev libssl-dev zlib1g-dev
```
Comment (lae) [inline docs/installation/Migrating-to-Python3]: Uninstalling development packages here and reinstalling them below doesn't quite make sense.

Comment (jeremystretch) [inline docs/installation/Migrating-to-Python3]: Agreed. Only the Python-specific packages should be listed.

Comment (vanderaaj) [inline docs/installation/Migrating-to-Python3]: I agree. In this process is that something I should then edit and re-submit a PR?

Comment (jeremystretch) [inline docs/installation/Migrating-to-Python3]: Yep, just push the new commit to your branch and the PR will get updated.

**Review by jvanderaa — COMMENTED** [review]
Updated to not to remove any dependency that would be re-installed.

**Pull request merged**